codescan/regexprs.go

```diff
@@ -81,7 +81,7 @@ var (
 	rxResponses       = regexp.MustCompile(`[Rr]esponses\p{Zs}*:`)
 	rxParameters      = regexp.MustCompile(`[Pp]arameters\p{Zs}*:`)
 	rxSchemes         = regexp.MustCompile(`[Ss]chemes\p{Zs}*:\p{Zs}*((?:(?:https?|HTTPS?|wss?|WSS?)[\p{Zs},]*)+)$`)
-	rxVersion         = regexp.MustCompile(`[Vv]ersion\p{Zs}*:\p{Zs}*(.+)$`)
+	rxVersion         = regexp.MustCompile(`^\s*[Vv]ersion\p{Zs}*:\p{Zs}*(.+)$`)
```

Comment:
I think this is an ad'hoc fix. This might fix your specific use-case but it breaks any attempt to consistently analyze tags. 

Why version should be treated in a special way, and other tags wouldn't have the ability to get leading blank space?

Wouldn't the solution to your issue rather lie around how to determine the boundaries of  the `swagger:meta`?

Reply:
I think the other meta tags could start with an optional space also. Currently, it just checks if any of the words are anywhere in the comment.  But they should be the first think on a line instead of in a sentence or part of another word.  I do realize this isn't a perfect solution as there may be some cases where you don't want to match one of these words on a line by itself. In my use case, it would be hard to determine the boundaries of `swagger:meta` as I have markdown docs right above the start of the tags.  Unless we had a start and end symbol to say these lines in between should be part of the `swagger:meta` yaml and nothing else.  Let me know what you would like to do. I can update the other regexes to include optional spaces if you think that would suffice.